index.html

```diff
@@ -1,14 +1,284 @@
+        illo.
+        Dignissimos?
+      </p>
+
+      <!-- Learn-More Article -->
+      <a href="/learn-more.html">
+        <h5 class="double-border-section">
+          GLOAT ONLY AFTER VICTORY
+        </h5>
+      </a>
+      <div class="pic-float">
+        <img src='/images/hexus.jpg' class="pic-shrink">
+      </div>
+      <p>
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptates adipisci porro consectetur exercitationem
+        corrupti facilis quod ipsa deleniti. Dolore, ea. At, necessitatibus saepe odio cupiditate enim ducimus ipsam
+        illo.
+        Dignissimos?
+      </p>
+      <a href="/learn-more.html"><button class="medium-button">
+          LEARN MORE
+        </button></a>
+
+      <!-- Learn-More Article -->
+      <a href="/learn-more.html">
+        <h5 class="higher-border">
+          UNDERATED WEOPONRY
+        </h5>
+      </a>
+      <div class="pic-float">
+        <img src='/images/Harley.webp' class="pic-shrink">
+      </div>
+      <p>
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptates adipisci porro consectetur exercitationem
+        corrupti facilis quod ipsa deleniti. Dolore, ea. At, necessitatibus saepe odio cupiditate enim ducimus ipsam
+        illo.
+        Dignissimos?
+      </p>
+      <button class="medium-button">
+        LEARN MORE
+      </button>
+    </section>
+  </section>
+
+  <!-- Contact Info Footer -->
+  <section id="footer">
+    <h5 class="centered-titles">
+      QUESTIONS OR COMMENTS? GET IN TOUCH:
+    </h5>
+
+    <!-- Footer Flex-Box -->
+    <section id="footer-box">
+
+      <!-- Footer Left Column -->
+      <section id="footer-left">
+        <input type="text" placeholder="Name" class="input" id="name-input">
+        <input type="text" placeholder="Email" class="input" id="email-input">
+        </div>
+        <section id="footer-bottom-left">
+          <textarea type="text" placeholder="Message" id="message-box"></textarea>
+          <button id="message-button">
+            SEND MESSAGE
+          </button>
+        </section>
+      </section>
+
+      <!-- Footer Right Column -->
+      <section id="footer-right">
+        <div id="contact-ipsum">
+          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Adipisci, culpa.
+        </div>
+
+        <!-- Social Media List -->
+        <div id="social-media">
+          <ul>
+            <li>
+              <img src="/images/illust58-6075-01.webp" class="pentagram"><span>545 Orange Grove Avenue,</span><br><span
+                id="orlando">Orlando, Florida</span>
+            </li>
+            <li id="villian-phone">
+              <img src="/images/illust58-6075-01.webp" class="pentagram"><span>666-666-6660</span>
+            </li>
+            <li id="villian-email">
+              <img src="/images/illust58-6075-01.webp" class="pentagram"><span>villainsAnonymous@gmail.com</span>
+            </li>
+          </ul>

```

Comment:
These three should be spaced out a little bit more, The bottom of this one should be in line with the bottom of the one on the right hand side. Also the Address in the given example was in the format of 3 lines where here we just have it occupying two